testCourse/courseInstances/Sp16/assessments/exam1-automaticTestSuite/infoAssessment.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Is the separate `Sp16` course instance necessary? If not, please remove it so we can keep focus on the relevant parts of this PR!

Reply:
No it's not. Just removed it!